conf/app.routes

```diff
@@ -1,3 +1,3 @@
 # microservice specific routes
 
-POST        /message             controllers.MessageController.handleMessageType
+POST        /messages             controllers.MessageController.handleMessageType
```

Comment:
@amaalali as we are no more handling message type, does the method name handleMessageType hold good ??
```suggestion
POST        /messages             controllers.MessageController.handleMessages
```